### ISSUE TITLE
 get_primary_company_address is undefined

### ISSUE THREAD

**Issue by swissbyte** [libracore/erpnextswiss#130]
Im using the following: 

ERPNext: v15.1.0
ERPNextSwiss: v1.23.1
Frappe Framework: v15.1.0

And it looks like that this function is missing for the salary certificate.
Was this a fuction built into older erpnext versions?

```
<table style="margin-left: 75px; width: 690px; height: 50px; margin-top: 32px; ">
  <!-- Footer -->
  {% set company = frappe.get_doc("Company", employee.company) %}
  {% set address = get_primary_company_address(employee.company) %}
  <tr>
    <td style="width: 230px; padding: 1px !important; "><br><br>
	{% if address.city %}{{ address.city or '' }},&nbsp;{% endif %}
```

**Comment by lasalesi** [comment]
This is a jinja function provided by ERPNextSwiss. Currently, there is no tested v15 support (still under development), so maybe the jenv import has failed in v15?

**Comment by lasalesi** [comment]
The jinja-hook has been renamed in v15 to jinja (rather than 'jenv`)

This is implemented in erpnextswiss:v15 with [22dc8e26](https://git.libracore.io/libracore/erpnextswiss/-/commit/22dc8e2676f1e43e466a77a0ec78229a6104b3c7)
